securedrop/secure_tempfile.py

```diff
@@ -3,7 +3,7 @@
 import os
 import io
 import six
-from tempfile import _TemporaryFileWrapper
+from tempfile import _TemporaryFileWrapper  # type: ignore
```

Comment:
noting for myself/others: this change is necessary is because `_TemporaryFileWrapper` is not in the type annotations for [Python 3's tempfile](https://github.com/python/typeshed/blob/master/stdlib/3/tempfile.pyi) upstream (but it is in [Python 2's tempfile](https://github.com/python/typeshed/blob/master/stdlib/2/tempfile.pyi#L20)) - will file issue upstream in a bit